cmd/workflow/controller/main.go

```diff
@@ -14,7 +14,7 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-var kubeConfigPath = flag.String("kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
+var kubeConfigPath = flag.String("kubeconfig", "/Users/caicloud/.kube/config", "Path to a kubeconfig. Only required if out-of-cluster.")
```

Comment:
What does `/Users/caicloud/.kube/config` stand for?

Reply:
It is my cube config for local test. I will remove it before commit.